context/portal.nginx.conf

```diff
@@ -1,3 +1,14 @@
+gzip on;
+gzip_disable "msie6";
+
+gzip_vary on;
+gzip_proxied any;
+gzip_comp_level 6;
```

Comment:
I haven't read deeply, but is it re-compressing on every request? There are also settings for server side caching, so it might be good to look at that, so the work only needs to be done once per file.

Reply:
I think I need to add the `gzip_static: on` and it will defer to using existing zipped assets where they exist.